python/graphstorm/eval/eval_func.py

```diff
@@ -195,13 +195,17 @@ def eval_roc_auc(logits,labels):
     # The roc_auc_score function computes the area under the receiver operating characteristic
     # (ROC) curve, which is also denoted by AUC or AUROC. The following returns the average AUC.
     rocauc_list = []
-    labels=labels_to_one_hot(labels, predicted_labels.shape[1])
```

Comment:
Can you add some unit test for this function?

Reply:
Do we have any existing unit test for binary classification?